postfix_exporter.go

```diff
@@ -663,16 +663,17 @@ func (e *PostfixExporter) Collect(ch chan<- prometheus.Metric) {
 
 func main() {
 	var (
-		listenAddress      = flag.String("web.listen-address", ":9154", "Address to listen on for web interface and telemetry.")
-		metricsPath        = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
-		postfixShowqPath   = flag.String("postfix.showq_path", "/var/spool/postfix/public/showq", "Path at which Postfix places its showq socket.")
-		postfixLogfilePath = flag.String("postfix.logfile_path", "/var/log/postfix_exporter_input.log", "Path where Postfix writes log entries. This file will be truncated by this exporter.")
-
-		systemdEnable                                 bool
+		app					= kingpin.New("postfix_exporter", "Prometheus metrics exporter for postfix")
```

Comment:
Why do you need to make this with New? Look [here](https://github.com/mjtrangoni/flexlm_exporter/blob/master/flexlm_exporter.go#L77) what I made. 
You could maybe add the short help variant, and the version too.
BTW, the indentation is not ok.